File: api/api/Models/fst/Section.cs
namespace api.Models.fst
{
    public class Section
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public bool IsUsed { get; set; } = true;
        public bool Hidden { get; set; } = false;

        public ICollection<Subsection> Subsections { get; set; } = new List<Subsection>();
    }
}
 

File: fst.kru/src/models/Section.ts
export interface Section {
  id: number;
  name: string;
  createdAt: Date;
  isUsed: boolean;
  hidden: boolean;

  subsections: Subsection[];
}

export interface Subsection {
  id: number;
  name: string;
  pdf: string;
  createdAt: Date;
  isUsed: boolean;
  hidden: boolean;

  sectionId: number;
  section: Section;
}
